### DISCUSSION TITLE
[Feature Request]: 下载目录及媒体目录改进申请

### DISCUSSION THREAD

**Discussion by BackspaceM** [jxxghp/MoviePilot#3471] [讨论]
### 当前程序版本

1.9.2-beta

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

1.9.1已增加了多目录支持，但可能由于多目录与媒体库目录存在遍历匹配的问题，会导致同类相近的文件夹系统健康体检报错（虽不影响使用，但有一些强迫症，加之考虑mp更好的发展），在此，本人有一些拙见，下载目录与媒体目录可用连线/磁吸解决，这样看起来简单不容易出错（图中箭头视为连线即可），还提升了很大的自由度（连线时可选择硬链接、软连接、复制等，在未有同级父文件夹时，硬链接变为灰色不可选）

PS：
此截图为刚更新，还未调整动漫媒体目录优先级（看公告说的很清楚，不知为啥很多人不理解）
另额外建议手动整理的刮削功能默认打开或者增加是否默认打开的选项(（虽选中目录会自动切换，但也会多一步），一般手动整理绝大多数为自动识别错误，手动整理时会额外多一步
加下载目录和媒体目录只是单纯想完美显示硬盘空间（单盘Basic模式）

![目录](https://github.com/jxxghp/MoviePilot/assets/74869843/62097bfb-841b-4d46-b8af-973fa2608d4c)
![报错](https://github.com/jxxghp/MoviePilot/assets/74869843/f08b7e2f-54c2-4764-bc78-f6096b5f7abe)
![改进建议](https://github.com/jxxghp/MoviePilot/assets/74869843/7bdcafbf-c634-447f-a053-b35b3045782c)

**Comment by huanyingbaye** [comment]
同求
